packages/paste-design-tokens/package.json

```diff
@@ -9,14 +9,23 @@
   "license": "MIT",
   "sideEffects": false,
   "files": [
-    "dist"
+    "dist",
+    "js",
+    "types"
   ],
   "scripts": {
-    "build": "yarn clean && gulp",
+    "build": "yarn clean && gulp && yarn build:js && tsc",
+    "build:js": "NODE_ENV=production node build.js",
     "build:dev": "yarn build",
-    "build:watch": "yarn clean && gulp dev",
+    "build:watch": "gulp dev",
     "clean": "rm -rf ./dist",
     "tokens": "gulp default",
     "test": "jest"
```

Comment:
Let's add the `"tsc": "tsc"` command here so that when we run `yarn type-check` in tests, it also runs the typechecker in this project: https://github.com/twilio-labs/paste/blob/main/package.json#L54

Reply:
Fair